lib/MultiModuleFactory.js

```diff
@@ -2,18 +2,18 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var Tapable = require("tapable");
-var MultiModule = require("./MultiModule");
+"use strict";
 
-function MultiModuleFactory() {
-	Tapable.call(this);
-}
-module.exports = MultiModuleFactory;
+const Tapable = require("tapable");
+const MultiModule = require("./MultiModule");
 
-MultiModuleFactory.prototype = Object.create(Tapable.prototype);
-MultiModuleFactory.prototype.constructor = MultiModuleFactory;
+module.exports = class MultiModuleFactory extends Tapable {
+	constructor() {
+		super();
+	}
```

Comment:
This constructor _looks_ redundant, but I guess it does prevent passing extra arguments to the parent constructor.